eox_nelp/one_time_password/view_decorators.py

```diff
@@ -57,7 +57,7 @@ def wrapper(request):
             if not proposed_user_otp == cache.get(user_otp_key):
                 return HttpResponseForbidden(reason="Forbidden - wrong code")
 
-            save_extrainfo_field(request.user, "is_phone_validated", True)
+            save_extrainfo(request.user, {"is_phone_validated": True})
```

Comment:
This works, even though the form doesnt have "is_phone_validated" field.
https://github.com/nelc/nelp-custom-registration-fields/blob/714b145/custom_reg_form/forms.py#L17
Django powers

Reply:
Indeed it's not working, the test is wrong and it's marking a false positive

Reply:
I added that to this pr https://github.com/nelc/nelp-custom-registration-fields/pull/14

Reply:
fixed